src/Timer.h

```diff
@@ -0,0 +1,22 @@
+#include <sys/time.h>
+#include <cstddef>
+#include <vector>
+
+class Timer {
+    timeval t_dummy;
+    double t_start;
+    double t_end;
+    double diff;
+    std::vector<std::vector<float>> predicted_times;
+
+    public:
+    Timer() : t_start(-1), t_end(-1), diff(-1) {}
+    Timer(std::vector<std::vector<float>> pt) : t_start(-1), t_end(-1),
+    diff(-1) {
```

Comment:
This is way too confusing. Use this style. https://github.com/NigoroJr/uccineering/blob/master/src/Node.cpp